.github/workflows/release.yaml

```diff
@@ -0,0 +1,43 @@
+name: Release draft creation
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+    - name: get latest release with tag
+      id: latestrelease
+      run: |
+          echo "::set-output name=releasetag::$(curl -s https://api.github.com/repos/OCTRI/authentication-lib/releases/latest | jq '.tag_name' | sed 's/\"//g')"
+    
+    - name: confirm release tag
+      run: |
+          echo ${{ steps.latestrelease.outputs.releasetag }}
+    
+    - name: tagcheckout
+      uses: actions/checkout@v3
+      with:
+          ref: ${{ steps.latestrelease.outputs.releasetag }}
+    
+    - name: Set up JDK 17
+      uses: actions/setup-java@v3
+      with:
+        java-version: '17'
+        distribution: 'temurin'
+        server-id: github
+
+    - name: Build with Maven
+      run: mvn package -DskipTests
```

Comment:
This step is redundant. When you run `mvn deploy`, the preceding phases of the lifecycle will run, which includes package.